cpr_data/cpr_derived_indices_data.sql

```diff
@@ -128,10 +128,10 @@ SELECT
        m.longitude AS "Longitude",
        m.sampledateutc AS "SampleDate_UTC",
        to_char(m.sampledatelocal, 'YYYY-MM-DD HH24:MI:SS') AS "SampleDate_Local",
```

Comment:
As per Claire's summary of our discussion on naming of date/time fields (https://github.com/aodn/public-schema/issues/43#issuecomment-1114426706), this is a full timestamp, so should be called "Sample**Time**_Local"

Same for the other CPR products below.
And if we can be bothered making that consistent, we should also change "SampleDate_UTC" to "SampleTime_UTC" (which will then have implications in geoserver-config filters, e.g. [cpr_phytoplankton_abundance_htg_data](https://github.com/aodn/geoserver-config/blob/67cbd1ac334654f9e503d6d42d98845dccbd0192/workspaces/imos/JNDI_imos_cpr_db/cpr_phytoplankton_abundance_htg_data/filters.xml#L11) )

@clairedavies What do you think?

Reply:
I think the consistency is important here, so yes, the changes should apply to UTC as well - as per #43

Reply:
Updated in {this commit](https://github.com/aodn/public-schema/pull/49/commits/eacb5e4743e89c17b390751a2d266a9c8f54c2d0)